### ISSUE TITLE
Support for MD+ command 1Fh (play wave file)

### ISSUE THREAD

**Issue by ArcadeTV** [ekeeke/Genesis-Plus-GX#503]
I'd like to ask if support for command 1Fh (play wave file) is already implemented in Genesis-Plus-GX.
I made a test-rom that is not guaranteed to work, because I don't have a MegaSD, but the expected output is a blank screen with MD+ CDDA music playing and a ring sfx coming from the r.wav file.

I've tested this with the "MegaSD" CD-Addon option set.

[mdplus_playground.zip](https://github.com/ekeeke/Genesis-Plus-GX/files/11311628/mdplus_playground.zip)
[MegaSD_DEV_Manual.pdf](https://github.com/ekeeke/Genesis-Plus-GX/files/11311641/MegaSD_DEV_Manual.pdf)

Thank you very much for this and your continued support of this great emulator!

**Comment by ekeeke** [comment]
No, support for commands that access disk space are not supported (see https://github.com/ekeeke/Genesis-Plus-GX/blob/master/core/cart_hw/megasd.c#L849 for the list of unsupported commands).

The reason was mainly that it requires knowing where files are stored, browse directories and read files from user storage, which is usually the goal of the frontend, not the emulation core (which is supposed to be multi platform).

**Comment by ekeeke** [comment]
>I still have to ask - would it help to make a rule like "has to be in the same folder as the rom file" and ignore any given path?

The spec supports both relative (from loaded ROM file folder) and absolute path for these commands because it is designed for use xith a sdcard , but yes, in the emulator, I could limit myself to support relative paths (paths that starts by "./" according to the spec) and do not allow access outside of loaded ROM directory (allowing access to any file on a computer is kinda problematic, even if it is only read access).

**Comment by ArcadeTV** [comment]
That's true - if it's not meant to be played simultaniously then it doesn't make sense.
Anyways, thank you very much for clearing!

**Comment by ekeeke** [comment]
Closing this for now as there is indeed not much interest in this command as you can simply do the same using a CUE file with audio tracks pointing to WAV files.

For enhanced sfx, I think a solution would be to use the PCM chip overlay through the enhanced mapper which is described in the Mega SD Dev Manual (and is emulated by Genesis Plus GX).

**Comment by ArcadeTV** [comment]
Thanks again for your feedback! 
Yes, I'd love to play with the method described in the manual, but I'm struggeling with a pcm playback routine for the sub cpu, I'm afraid I'm totally lost without some source code to get me started. 
Anyways, thank you.
